javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java

```diff
@@ -99,6 +100,9 @@
 
     @InternalProperty
     private Storage storage;
+    
+    // printer used to print the node
+    private Printable printer;
```

Comment:
A per-node printer? Do we want to specifiy printers at this detailed level? Is this still easy to use?